### PR TITLE
Update templates and integrate with slack

### PR DESCRIPTION
**Pull request by juwara0**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [ ] #none# - documentation fixes and/or test additions
- [X] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

Closes: https://agile-jira.ciena.com/browse/FROST-1281 and https://agile-jira.ciena.com/browse/FROST-1320

# CHANGELOG
* **Updated** pull request template
* **Added** issue template
* **Updated** to `pr-bumper` version `3`
* **Updated** to node 8
* **Added** slack integration
* **Updated** `ember-frost-test` to `^4.0.1`
* **Updated** `ember-test-utils` to `^8.1.1`
* **Updated** `ember-cli-frost-blueprints` to `^5.0.2`
* **Added** `package-lock.json` file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,11 +42,13 @@ matrix:
 
 before_install:
 - npm config set spin false
-- npm install -g coveralls pr-bumper@^1.0.0
+- npm install -g bower
```
Comment (notmessenger) [inline .travis.yml]: Why is this necessary when Bower is listed as a _devDependency_?  Are we going to have to add it to other repos too that don't currently have this entry?

Comment (juwara0) [reply .travis.yml]: The build was failing and I am in the middle of determining if I need to `npm install -g bower` and do `bower install` in travis CI or just run `bower install`

Comment (juwara0) [reply .travis.yml]: @notmessenger - Alright, the build is passing now with just running `bower install` in travis.

**Comment by notmessenger** [comment]
👍
